worlds/stardew_valley/strings/animal_product_names.py

```diff
@@ -3,17 +3,32 @@ class AnimalProduct:
     brown_egg = "Egg (Brown)"
     chicken_egg = "Chicken Egg"
     cow_milk = "Cow Milk"
+    dinosaur_egg_starter = "Dinosaur Egg (Starter)"
+    """This item does not really exist and should never end up being displayed. 
+    It's there to patch the loop in logic because of the Dinosaur-and-egg problem."""
     dinosaur_egg = "Dinosaur Egg"
+    duck_egg_starter = "Duck Egg (Starter)"
+    """This item does not really exist and should never end up being displayed. 
+    It's there to patch the loop in logic because of the Chicken-and-egg problem."""

```

Comment:
Duck-and-egg problem?

Reply:
Pretty sure the chicken came before the duck